### ISSUE TITLE
Real-time implementation of multi-source sky models

### ISSUE THREAD

**Issue by danasimard** [dsa110/dsa110-issues#64]
Issue depends on issue #57 

Implement & benchmark real-time generation of sky models and calibration.

Antenna pointing feedback in grafana.

**Comment by danasimard** [comment]
Closing, as this is no longer part of the requirements for real-time calibration.  Multi-source models may still be needed for direction-dependent calibration of the candidate images, but that should be a separate issue.
